website/src/components/HomepageFeatures.js

```diff
@@ -37,7 +37,7 @@ const FeatureList = [
     link: "https://github.com/Azure/awesome-azd/issues/new/choose",
     description: (
       <>
-        Consider sharing your template in our <a href="/gallery">Gallery</a> to help the community. Just <a href="https://github.com/Azure/awesome-azd/issues/new/choose">fill in the issue </a> and we'll do the rest!
+        Consider sharing your template in our <a href="/awesome-azd">Gallery</a> to help the community. Just <a href="https://github.com/Azure/awesome-azd/issues/new/choose">fill in the issue </a> and we'll do the rest!
```

Comment:
the app does not have an "/awesome-azd" route - did you mean to use "/" (so it points to the default homepage for Gallery?)